apps/test/unit/lib/tools/jsdebugger/DebugConsoleTest.js

```diff
@@ -236,3 +237,137 @@ describe('The DebugConsole component', () => {
     });
   });
 });
+
+describe('The DebugConsole component with the react-inspector flag on', () => {
+  let root;
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers(reducers);
+    sinon.stub(experiments, 'isEnabled').returns(true);
+    expect(experiments.isEnabled('react-inspector')).to.equal(true);
+
+    root = mount(
+      <Provider store={getStore()}>
+        <DebugConsole showReactInspector={true} />
+      </Provider>
+    );
+  });
+
+  afterEach(() => {
+    experiments.isEnabled.restore();
+    restoreRedux();
+  });
+
+  const debugOutput = () => root.find('#debug-output');
+
+  describe('when input is coming from code workspace returns', () => {
+    it('an array input outputs an array without an arrow', () => {
```

Comment:
Is there supposed to be no arrow in line 272?

Reply:
I found "input outputs" to be a bit confusing. But I was having a hard time thinking of an alternative. Perhaps something like one of these?
- a logged array prints an array with an arrow
- a parsed array returns an expandable array
- debug output for an array includes an expander icon

Reply:
for the describe (again, I'm not sure what the best alternative is), I found "returns" a bit confusing since it can be either a noun or a verb and I wasn't sure which it was. Possible alternatives:
- when input originates from code workspace console.log
- with explicit call to console.log

Reply:
1. Oh right! Because it has an expandable arrow, I should say with an arrow! Totally missed to mention that! Thanks for catching that! 
2. I'll refactor it to be `a logged array prints an array with an expander icon`
3. `when input originates from code workspace console.log` sounds much better than what I have there! I'll refactor it to that.